src/Activation/AcceptEula.tsx

```diff
@@ -63,7 +64,9 @@ const AcceptEula: FunctionComponent = () => {
             docName={"onboarding.privacy_policy"}
```

Comment:
Should this component be named `EulaLink` or just something like `Link` because it handles the Eula and Privacy Policy links?